frontend/packages/data-portal/app/components/Layout/TopNavigation.tsx

```diff
@@ -31,6 +31,13 @@ export function TopNavigation() {
     ? TOP_LEVEL_LINKS
     : TOP_LEVEL_LINKS_COMPETITION
 
+  const isItNeuroglancerPage = pathname.includes("/view/runs/");
+
+  if(isItNeuroglancerPage) {
```

Comment:
nit: if with space, so if (isItNeuroglancerPage)

Reply:
I didn't get this point
Should i replace is with ifItIsNeuroglancerPage? @seankmartin